examples/elasto_plastic.jl

```diff
@@ -0,0 +1,300 @@
+using JuAFEM
+using Tensors
+using KrylovMethods
+using TimerOutputs
+using Statistics: mean
+import ProgressMeter
+const ∇ = Tensors.gradient
+
+struct Material{T}
+    μ::Float64		#Lamé constant
+    λ::Float64		#Lamé constant
+    τ₀::Float64		#Yield stress
+    H::Float64		#Hardening
+end
+
+struct MaterialState{T}
```

Comment:
Same with tabs here.

If you don't want to hardcode the dimension you could do something like

```jl
struct MaterialState{T, dim, ST <: SymmetricTensor{2, dim, T}, TT <: Tensor{2, dim, T}}
S::ST
F⁻¹::TT
...
end
```
```